src/doc/gui_manual/Preferences/File_Locations.rst

```diff
@@ -0,0 +1,271 @@
+the user may need to be aware of the location of these files on either the client
+or the server or both.
+
+Files in ``VUSER_HOME``
+~~~~~~~~~~~~~~~~~~~~~~~
+Most of the files associated with VisIt_ configuration have their prescribed home
+in ``VUSER_HOME``. When running in client/server, it is the configuration files
+on the *local client* that effect behavior. This means they are always the
+files on the *local* machine and not the *remote* system. Any configuration
+files that might also be on the remote server do not play a role in effecting
+behavior in client/server mode.
+
+Settings/Preferences File
+"""""""""""""""""""""""""
+* Location and file name: ``VUSER_HOME/config`` 
+* Purpose: Holds user settings from :ref:`preferences window <Preferences Window>`
+  plus numerous other settings such as default attributes for operators and plots,
+  default database read options, default color tables, as well as the
+  enabled/disabled state of various plot, operator and database plugins.
+* Written: when user :ref:`saves settings <How to Save Settings>`.
+* Read: on VisIt_ startup but this can be overridden by the ``-noconfig``
+  command-line :ref:`startup option <StartupOptions>`.
+* Format: ASCII `XML <https://en.wikipedia.org/wiki/XML>`_
+
+GUI Configuration File
+""""""""""""""""""""""
+* Location and file name: ``VUSER_HOME/guiconfig``
+* Purpose: Holds positions and sizes of various GUI windows. Also holds the list of
+  recently used paths to open databases.
+* Otherwise operates identically to ``VUSER_HOME/config``.
+
+Host Profile Files
+""""""""""""""""""
+* Location and file name(s): ``VUSER_HOME/host/host_<site-name>_<resource-name>.xml``
+* Purpose: Stores information on how to connect to and launch jobs on various
+  compute resources. In many cases, there are separate sets of host profile files
+  for all the compute resources at a commonly used site such as LLNL CZ or RZ,
+  ANL, ORNL, etc. Often sites will *post* VisIt_ host profile files in places for
+  users to easily find and *install* them. Installing them is just a matter of
+  copying them to ``VUSER_HOME``.
+* Written: when user :ref:`saves settings <How to Save Settings>`.
+* Read: on VisIt_ startup. All host profiles in ``VUSER_HOME/host*.xml`` are read
+  on VisIt_ startup but this can be overridden by ``-noconfig``. Users should be
+  aware of this behavior. If the user passes ``-noconfig`` for the purposes of
+  avoiding the loading of preferences, s/he will also be without any host profiles.
+* Format: ASCII `XML <https://en.wikipedia.org/wiki/XML>`_
+
+.. _file_locations_visitrc:
+
+VisIt Run Commands (rc) File
+""""""""""""""""""""""""""""
+* Location and file name: ``VUSER_HOME/visitrc``
+* Purpose: Holds Python code to be executed each time VisIt_ is launched.
+* Written: Whenever user hits the **Update Macros** button in the
+  :ref:`**Command Window** <Command_Line_Window>`.
```

Comment:
"**Command Window**" is showing the "**" in the html output and not making the text bold as I assume you intended.